### PR TITLE
Add changelog for 2.24.0

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:
This is the final changelog for v2.24.0.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: docs/changelogs/CHANGELOG-2.24.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- [v2.24.0](#v2240)
+
+## [v2.24.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.24.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.24/installation/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure user cluster reconciliations will not cause resource exhaustion on seed clusters. A [full upgrade guide is available from the official documentation](https://docs.kubermatic.com/kubermatic/v2.24/installation/upgrading/upgrade-from-2.23-to-2.24/).
+
+### Read Before Upgrading
+
+- **ACTION REQUIRED:** legacy backup controller has been removed. Before upgrading, please change to the backup and restore feature that uses [backup destinations](https://docs.kubermatic.com/kubermatic/v2.24/tutorials-howtos/etcd-backups/) if the legacy controller is still in use ([#12473](https://github.com/kubermatic/kubermatic/pull/12473))
+* `s3-storeuploader` has been removed ([#12473](https://github.com/kubermatic/kubermatic/pull/12473))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: Why is this a `*` and not a `-`? :-D

File: docs/changelogs/CHANGELOG-2.24.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+- OpenVPN for control plane to node connectivity has been deprecated. It will be removed in future releases of KKP. Upgrading all user cluster to Konnectivity is strongly recommended ([#12691](https://github.com/kubermatic/kubermatic/pull/12691))
+
+### API Changes
+
+- The field `vmNetName` in `Cluster` and `Preset` resources for vSphere clusters is deprecated and `networks` should be used instead ([#12444](https://github.com/kubermatic/kubermatic/pull/12444))
+- The field `konnectivityEnabled` in `Cluster` resources is deprecated. Clusters should set this to `true` to migrafe off OpenVPN as Konnectivity being enabled will be assumed in future KKP releases ([#12691](https://github.com/kubermatic/kubermatic/pull/12691))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: migrafe => migrate

File: docs/changelogs/CHANGELOG-2.24.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+- Extend `kubermatic-installer mirror-images` command with an option to export a tarball instead of syncing to a remote repository. This can be helpful in airgapped scenarios ([#12613](https://github.com/kubermatic/kubermatic/pull/12613))
+- Extend MinIO configuration options to allow enabling MinIO console access and exposing MinIO API and console via Ingress ([#12683](https://github.com/kubermatic/kubermatic/pull/12683))
+- New configuration option for Dex (`oauth` chart): Allow modification of web frontend issuer ([#12608](https://github.com/kubermatic/kubermatic/pull/12608))
+- Support for configuring IPFamilies and IPFamilyPolicy for nodeport-proxy ([#12472](https://github.com/kubermatic/kubermatic/pull/12472))
+- Support for configuring OIDC username and group prefix for user clusters ([#12648](https://github.com/kubermatic/kubermatic/pull/12648))
+- Support for configuring the dex theme via values file ([#12560](https://github.com/kubermatic/kubermatic/pull/12560))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: capitalize Dex

File: docs/changelogs/CHANGELOG-2.24.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+- Support to configure node address type for kubelb at the datacenter level ([#12715](https://github.com/kubermatic/kubermatic/pull/12715))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: It's **KubeLB**, never kubelb or kubeLB or kubeLb or Kubelb or KubelB.

File: docs/changelogs/CHANGELOG-2.24.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+- Update etcd for user clusters to 3.5.9 ([#12453](https://github.com/kubermatic/kubermatic/pull/12453))
+- Update KubeVirt chart for the installer local command to the latest release v1.0.0 ([#12470](https://github.com/kubermatic/kubermatic/pull/12470))
+- Update metering Prometheus to next LTS version 2.45.0 ([#12532](https://github.com/kubermatic/kubermatic/pull/12532))
+- Update metrics-server for all deployments to v0.6.4 ([#12516](https://github.com/kubermatic/kubermatic/pull/12516))
+- Update nginx-ingress-controller to v1.9.3 (fixes CVE-2023-44487, HTTP/2 rapid reset attack) ([#12712](https://github.com/kubermatic/kubermatic/pull/12712))
+- Update supported Kubernetes releases for EKS/AKS ([#12579](https://github.com/kubermatic/kubermatic/pull/12579))
+- Update telemetry-agent to v0.4.1 ([#12572](https://github.com/kubermatic/kubermatic/pull/12572))
+- Update to controller-runtime 0.16.1 / Kubernetes 1.28 ([#12609](https://github.com/kubermatic/kubermatic/pull/12609))
+- Update to Go 1.21.3 ([#12697](https://github.com/kubermatic/kubermatic/pull/12697))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: `Update to <library>` => `Update <library> to`, then re-sort all the update bulletpoints.

File: docs/changelogs/CHANGELOG-2.24.md (hunk 1 above)
Comment (embik) [reply docs/changelogs/CHANGELOG-2.24.md]: Because of reasons!

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve
/hold
if other folks want to take a look as well

**Comment by embik** [comment]
/hold cancel

Let's go ahead with the release.

**Pull request merged**